docs/literate/src/files/first_steps/create_first_setup.jl

```diff
@@ -0,0 +1,245 @@
+
+# The linear scalar advection equation is already implemented in Trixi.jl as
+# [`LinearScalarAdvectionEquation2D`](@ref), for which we need to define a two-dimensional parameter
+# with the `advection_velocity`. Appropriate for our problem is `(0.2, -0.7)`.
+
+advection_velocity = (0.2, -0.7)
+equations = LinearScalarAdvectionEquation2D(advection_velocity)
+
+# To solve our problem numerically using Trixi.jl, we have to discretize the spatial
+# domain, for which we set up a mesh. One of the most used meshes in Trixi.jl is
+# [`TreeMesh`](@ref). The spatial domain used is ``[-1, 1] \times [-1, 1]``. We set an initial number
+# of elements in the mesh using `initial_refinement_level`, which describes the initial number of
+# hierarchical refinements. The variable `n_cells_max` is used to limit the number of elements in the mesh,
+# which cannot be exceeded when using [adaptive mesh refinement](@ref Adaptive-mesh-refinement).
+
+# All minimum and all maximum coordinates must be combined into `Tuples`.
+
+coordinates_min = (-1.0, -1.0)
+coordinates_max = ( 1.0,  1.0)
+mesh = TreeMesh(coordinates_min, coordinates_max,
+                initial_refinement_level = 4,
+                n_cells_max = 30_000)
+
+# To approximate the solution of the defined model, we create a [`DGSEM`](@ref) solver.
+# The solution in each of the recently defined mesh elements will be approximated by a polynomial
+# of degree `polydeg`. For more information about discontinuous Galerkin methods,
+# check out the [Introduction to DG methods](@ref scalar_linear_advection_1d) tutorial.
+
+solver = DGSEM(polydeg=3)
+
+# Now we need to define an initial condition for our problem. All the already implemented
+# initial conditions for [`LinearScalarAdvectionEquation2D`](@ref) can be found in
+# [`src/equations/linear_scalar_advection_2d.jl`](https://github.com/trixi-framework/Trixi.jl/blob/main/src/equations/linear_scalar_advection_2d.jl).
+# If you want to use, for example, a Gaussian pulse, it can be used as follows.
+# ```julia
+# initial_conditions = initial_condition_gauss
+# ```
+# But to show you how an arbitrary initial condition can be implemented in a way suitable for
+# Trixi.jl, we define our own initial conditions.
+# ```math
+# u(0,x,y) = \sin\bigl(\pi x \bigr) \sin\bigl(\pi y \bigr)
+# ```
+# The initial conditions function must take coordinates, time and equation as arguments
+# and return an initial condition as a static vector `SVector`. Following the same structure, you
+# can define your own initial conditions.
+
+function initial_condition_sinpi(x, t, equations::LinearScalarAdvectionEquation2D)
+    scalar = sinpi(x[1]) * sinpi(x[2])
+    return SVector(scalar)
+end
+initial_condition = initial_condition_sinpi
+
+# The next step is to define a function of the source term corresponding to our problem.
+# ```math
+# f(t,u,x,y) = - 2 e^{-t} \sin\bigl(2 \pi (x - t) \bigr) \sin\bigl(2 \pi (y - t) \bigr)
```

Comment:
```suggestion
# f(u, x, y, t) = - 2 e^{-t} \sin\bigl(2 \pi (x - t) \bigr) \sin\bigl(2 \pi (y - t) \bigr)
```
Maybe change the order of arguments to reflect the order in the source term function?